stable/jenkins/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: jenkins
 home: https://jenkins.io/
-version: 1.3.3
+version: 1.3.4
```

Comment:
1.3.4 has been released in the meantime. Please update to 1.3.5 and ensure that it can be merged into master.

Reply:
I've changed the version number, and rebased the chart on the latest master.